platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -61,7 +59,7 @@
 import org.powermock.modules.junit4.rule.PowerMockRule;
 
 @RunWith(MockitoJUnitRunner.class)
-@PrepareForTest({ConfigurationAdminMigration.class, FileUtils.class})
+@PrepareForTest({ConfigurationAdminMigration.class, FileUtils.class, Files.class})
 public class ConfigurationAdminMigrationTest {
 
     private static final Path CONFIG_FILE_PATH = Paths.get("/root/etc/exported/etc/pid.config");
```

Comment:
updated code to:
private static final String CONFIG_FILE_PATH1 = CONFIG_PID + "1" + configurationFileExtension;
private static final String CONFIG_FILE_PATH1 = CONFIG_PID + "1" + configurationFileExtension;